src/Reactor.jl

```diff
@@ -69,6 +69,12 @@ function Reactor(domains::T,y0s::W,tspan::W2,interfaces::Z=Tuple(),ps::X=DiffEqB
                 domain.rxnarray[i,j] += k-1
             end
         end
+        for i in 1:length(domain.constantspeciesinds)
+            domain.constantspeciesinds[i] += k-1
+        end
+        for (thermovar,ind) in domain.thermovariabledict
+            domain.thermovariabledict[thermovar] += k-1
+        end
```

Comment:
Are these properties always used in a global indexing sense (as opposed to a domain indexing sense)?

Reply:
These were originally created to retrieve the index of thermovariables quickly when calculating adjoint sensitivity, and they are currently only used in `getadjointsensitivities` in a global indexing sense